docs/pages/blog/lab-tree-view-to-mui-x.md

```diff
@@ -0,0 +1,83 @@
+---
+title: The Tree View is moving to MUI X
+description: Migrate to the new package to start building with our powerful Tree View, now part of MUI X. Previously released MIT components will stay MIT.
+date: 2023-08-21T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+card: true
+---
+
+After more than 4 years in the lab, the [Tree View](https://mui.com/x/react-tree-view/) components have found a new home as part of MUI X.
+This means we'll be dedicating even more time and effort to these complex components, to better meet the needs of both you and your users.
+
+## TL;DR
+
+- The Tree View is one step closer to a stable release.
+- **No surprise licenses changes**. We are staying true to [our promises](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd).
+  What's MIT stays MIT.
+  Therefore, all the existing features and future features of the Tree View will remain MIT and free to use.
+- Follow the [migration steps](/x/migration/migration-tree-view-lab/)
+
+## What is the Tree View?
+
+The Tree View is a component to represent hierarchical data presented as nodes in a tree-like format.
+
+The component allows to select one or multiple nodes.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop>
+  <source src="/static/blog/lab-tree-view-to-mui-x/treeview.mov" type="video/mp4">
+</video>
+
+## What is MUI X?
+
+[MUI X](/x/) is a collection of advanced components built for complex use cases.
+
+As opposed to the MUI Core library, which leans on the open-source community for support, MUI X components require several full-time developers dedicated to engineering and ongoing maintenance.
+
+MUI X components are available under two licenses:
+
+- MUI X is MIT licensed, so it's free to use.
+- MUI X Pro includes the most advanced features and is commercially licensed.
+
+## Why did the Tree View move to MUI X?
```

Comment:
I think one of the reasons we could raise is that we want to go beyond what's offered in the MIT space.

I would perhaps argue like this:

"Once a component is ready to leave the lab, it can either go to MUI X or MUI Core libraries. 
Given the nature of the Treeview components, it was a clear choice to group it with the other advanced components. 
Besides, we plan to improve it beyond what's offered in the MIT space with extra fine-tuned features for enterprise and data-rich applications."